hudi-spark-datasource/hudi-spark/src/test/scala/org/apache/spark/sql/hudi/command/index/TestSecondaryIndex.scala

```diff
@@ -92,7 +92,15 @@ class TestSecondaryIndex extends HoodieSparkSqlTestBase {
             Seq("record_index", "record_index", "")
           )
 
-          spark.sql(s"create index idx_name on $tableName using secondary_index(name)")
+          // Secondary index can not be created for two columns at once
+          checkException(s"create index idx_name_price on $tableName using secondary_index(name,price)")(
```

Comment:
Can we add some positive scenario for all indexes and validate that e2e from  creation to udpate to drop works all good?

Reply:
Yes, we have covered those scenarios in TestSecondaryIndex(Test Create/Show/Drop Secondary Index), TestSecondaryIndexPruning(testCreateAndDropSecondaryIndex), TestExpressionIndex (Test Drop Expression Index) and TestIndexSyntax(Test Create/Drop/Show/Refresh Index, Test Create and Drop Index Syntax with Simple record key, Test Create and Drop Index Syntax with Complex record key.